app/src/main/java/homer/api/Controller.java

```diff
@@ -11,6 +11,13 @@ public interface Controller {
      */
     void connectDevice(Device<?> device);
 
+    /**
+     * Disconnects the device from the controller.
+     * 
+     * @param deviceId The ID of the device to be disconnected.
+     */
+    void disconnectDevice(long deviceId);
```

Comment:
Is it really necessary?  I think it'd be overkill, we'll be managing at most a few dozens devices, let's hear from @michelececcacci

Reply:
It's not about the number of devices, but hiding the primitive type and better conveying the notion.
It would allow us to change it to eg. a byte later on if we really want to, without impacting everything else.

Reply:
Alright then, what DS is more suitable?

Reply:
I was thinking just a class implementing compareTo/equals, and a method to serialize it as a string.
The constructor could contain the random generation. The value itself would be kept private.

Reply:
LGTM

Reply:
As discussed internally this will be revised later.